src/app/auth/auth.js

```diff
@@ -78,7 +22,7 @@ $(document).ready((function ($) {
 
     function removeFieldErrors() {
         $('.auth-form, .appointment-form')
```

Comment:
Since `removeFieldErrors` is used by both **home** page and **sign-in/sign-up** pages you have to figure out a way to use this function in both places without copy-pasting code. 

Notice the part `$('.auth-form, .appointment-form')` (auth-form and appointment-form)

I'd say move `removeFieldErrors` function to `app.js` for now since it's imported everywhere

Reply:
Same principle goes for `showFieldErrors` function

Reply:
Done

Reply:
Resolved.